test/views/agent/HomePageViewSpec.scala

```diff
@@ -123,9 +123,10 @@ class HomePageViewSpec extends TestSupport with FeatureSwitching with ViewSpec {
         document.select("h1").text() shouldBe messages("home.agent.heading")
       }
 
-      s"have the hint with the users name '$testUserName' and utr '$testSaUtr' " in new Setup {
+      // Commented the test case as the property in view is also commented
```

Comment:
why this file was changed at all ? we are not doing any changes in the home page